catalog/resourcemanagement/resourcemanagement-usage-ui/src/main/webapp/js/model/DataUsage.js

```diff
@@ -0,0 +1,173 @@
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*jshint browser: true */
+
+define(['backbone',
+        'jquery',
+        'underscore'],
+    function (Backbone, $, _) {
+
+        var DataUsage = {};
+
+        DataUsage.UsageModel = Backbone.Model.extend({
+
+            initialize: function() {
+                this.set({'users' : []});
+                this.set({'saving' : false });
+                this.getUsageData();
+                this.getCronTime();
+            },
+            getUsageData: function() {
+                console.log("getUsageData");
+                var url = "/jolokia/exec/org.codice.ddf.resourcemanagement.usage.service.DataUsage:service=datausage/userMap/";
+                var that = this;
+                $.ajax({
+                    url: url,
+                    dataType: 'json',
+                    success: function(data) {
+                        that.set({'users' : that.parseDataModel(data.value)});
+                    }
+                });
+            },
+            getCronTime: function() {
+                var url = "/jolokia/exec/org.codice.ddf.resourcemanagement.usage.service.DataUsage:service=datausage/cronTime/";
+                var that = this;
+                $.ajax({
+                    url: url,
+                    dataType: 'json',
+                    success: function(data) {
+                        that.set({'cronTime' : data.value });
+                    }
+                });
+            },
+            updateCronTime: function(time) {
+                var url = "/jolokia/exec/org.codice.ddf.resourcemanagement.usage.service.DataUsage:service=datausage/updateCronTime/" + time;
+                var that = this;
+                $.ajax({
+                    url: url,
+                    dataType: 'json',
+                    success: function() {
+                        that.getCronTime();
+                    }
+                });
+            },
+            parseDataModel: function(data) {
+                var dataModel = [];
+                var that = this;
+
+                 _.object(_.map(data, function (value, key) {
+
+                    var dataUsage = value[0];
+                    var dataLimit = value[1];
+                    var usagePercent = Math.round((dataUsage / dataLimit) * 100);
+                    if(usagePercent > 100) {
+                        usagePercent = 100;
+                    }
+
+                    var usageRemaining = that.constructUsageRemainingString(dataLimit, dataUsage);
+
+                    var displayLimit;
+                    var displaySize;
+
+                    if(dataLimit >= (1000 * 1000 * 1000)) {
```

Comment:
i've always used 1024 to represent a megabyte, but I guess according to wikipedia and what they submitted to the international system of quantities it should be 1000. So 1000 should be correct. Unless you meant something else